### ISSUE TITLE
Separate instances to multiple Hass devices

### ISSUE THREAD

**Issue by kpine** [cgarwood/homeassistant-zwave_mqtt#63]
- Home Assistant version: 0.105.5
- ozwdaemon version or add-on version: 0.1
- homeassistant-zwave_mqtt version:  0.0.5

The native zwave integration represents multi-channel zwave devices as multiple Home Assistant Devices, where the root endpoint (if applicable) and each endpoint are individual Devices. I am assuming you want to duplicate this behavior.

Attached is a dump with a thermostat that has two instances. Topics are, for example:
```
OpenZWave/1/node/9/instance/1
OpenZWave/1/node/9/instance/2
```
[mqtt_dump.txt](https://github.com/cgarwood/homeassistant-zwave_mqtt/files/4233030/mqtt_dump.txt)

**Comment by marcelveldt** [comment]
No, a single device will be represented as one single device (=piece of hardware).
But it may/will have multiple entities.

**Comment by kpine** [comment]
Why is that changing for this implementation?

**Comment by marcelveldt** [comment]
Bacause the device is still one device ?
Please explain your usecase for having one single z-wave device split up into multiple so we can understand.

**Comment by marcelveldt** [comment]
Got it. We're talking about devices with multiple instances/channels and the fact that each instance can be in a different room/area. The current implementation actually splits those instances into multiple devices in Home Assistant. 

This new implementation does not yet do that. We leave this here as feature request.

**Comment by marcelveldt** [comment]
@kpine am I correct in concluding that your use case for this is that instances of each device can be in another room/area ?

**Comment by kpine** [comment]
It's not my use case, that's the justification your fellow developers in this project, and other HA Z-Wave developers, have put forth in the past. So you'll need to ask them. 😉 This just caught my attention because there have been at least two issues submitted to HA regarding this behavior, and they were closed with the statement that separate HA Devices for multi-channel Z-Wave devices is the designed behavior and will not change. Given that history, when I noticed my multi-channel thermostat was a single device in this implementation, and two devices in native zwave, I figured I'd log the issue so it wouldn't get missed. 🤷‍♂ In my case, the second device is useless actually.

**Comment by marcelveldt** [comment]
Yeah, I'm informed now ;-)
And I can confirm the behaviour you've spotted. In my case the second device is useless too in 99% of the cases. But in the scenario where you have for example a double relay and one endpoint of that relay is controlling some device in the garden and the other in the garage, you can't assign that device to only the garden or the garage if you want to assign it to rooms.

**Comment by marcelveldt** [comment]
Fixed in #92
